src/main/java/mekanism/common/tile/transmitter/TileEntitySidedPipe.java

```diff
@@ -401,13 +403,20 @@ protected void onRefresh() {
     }
 
     public void refreshConnections() {
-        if (redstoneReactive) {
-            redstonePowered = MekanismUtils.isGettingPowered(getWorld(), new Coord4D(getPos(), getWorld()));
-        } else {
-            redstonePowered = false;
-        }
+        if (handlesRedstone()) {
+            boolean previouslyPowered = redstonePowered;
+            if (redstoneReactive) {
+                redstonePowered = MekanismUtils.isGettingPowered(getWorld(), new Coord4D(getPos(), getWorld()));
+            } else {
+                redstonePowered = false;
+            }
+            //If the redstone mode changed properly update the connection to other transmitters/networks
+            if (previouslyPowered != redstonePowered) {
+                markDirtyTransmitters();
```

Comment:
Probably better to directly call `notifyNeighbours`

Reply:
Now calls `notifyTileChange()` which seems to be a helper method to keep it cleaner when notifying neighbors.